website/docs/faqs/Core/_category_.yaml

```
@@ -0,0 +1,10 @@
+# position: 2.5 # float position is supported
+label: 'Core'
```

Might be a good opportunity to update this title on the sidebar
```suggestion
label: 'dbt Core'
```